Lesson4/src/com/home/model/MilitaryOffice.java

```diff
@@ -0,0 +1,48 @@
+package com.home.model;
+
+import com.home.service.PersonsRegistry;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class MilitaryOffice {
+    PersonsRegistry registry;
+
+    public MilitaryOffice(PersonsRegistry registry) {
+        this.registry = registry;
+    }
+
+    public List<Person> getListOfHealthyMen(Address address) {
+        List<Person> listOfMen = new LinkedList<>();
+        for (Person person : registry.getListOfPeople(address)) {
+            int age = person.getAge();
+            if (person.getGender().equals("male") && (age >= 18 && age <= 27)) {
+                listOfMen.add(person);
+            }
+        }
+        return listOfMen;
+    }
+
+    public List<Person> getListOfHealthyMen(String country) {
+        List<Person> listOfMen = new LinkedList<>();
+        for (Person person : registry.getListOfPeople(country)) {
+            int age = person.getAge();
+            if (person.getGender().equals("male") && (age >= 18 && age <= 27)) {
+                listOfMen.add(person);
+            }
```

Comment:
since you have the same check here and in the method above I'd recommend moving it to a separate method that would return boolean and use here and in the method above